app/commands/scrape_command.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class ScrapeCommand
+  attr_reader :keyword
+
+  def initialize(keyword_id:)
+    @keyword_id = keyword_id
+  end
+
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
```

Comment:
I'm not sure how to address these two lint rules. IMO, this method is simple enough to a human to parse/understand.

I would prefer not to extract the rescue part to separate functions as it fragments the method.

Reply:
The method is too long because of the double `rescue` block and logging statements. While it is readable, there is complexity in the error handling, that's the code smell that Rubocop is raising. Using `rescue` blocks should be limited to maximum one (these are not comparable to functional paradigms like `case/else` or `with/else` in Elixir). To achieve this, it is best to raise one type of error, not multiple.

Reply:
Fixed in b0713248018bbc87ba507fb4a5ffa388a807a3df